README.md

```diff
@@ -1,24 +1,161 @@
 # Rails Dynamic Request
 
-## Objectives
+## Review
 
+You already know how to create a static request, which is where you create a page that doesn't take any parameters and simply renders a view, an example would be: ```localhost:3000/about```. For Rails to process this request, the ```route.rb``` file contains a route such as:
 
-1. Explain the differences between a static route and a dynamic route with regards to variables
-2. Write a dynamic route with the :variable syntax
-3. Access a route variable value from params
-4. Use a route variable value in params within a controller action (like in Post.find(params[:id])
-5. Use the convention of naming the primary key :id as a route variable
-6. Pass an instance variable to the view
-7. use a route variable to load a object and render that show page
+```ruby
+get 'about', to "static#about"
+```
 
-## Notes
+This is mapped to the ```static``` controller and ```about``` action, which renders the ```about.html.erb``` view template.
 
-The next restful action we want to build is the ability to show a post.
+## Dynamic Requests
 
-to do that we need to integrate route variables into our routes for a post /posts/1 can't be hardcoded.
+If you are comfortable with how static requests you will pick up on dynamic requests quickly. An example scenario for a dynamic request would be where a user navigates to a page in the application such as ```localhost:3000/posts/42``` and is displayed a post with the id of 42. Behind the scenes what is happening is below:
```

Comment:
I would connect this back to the RESTful routes stuff. So "We according to REST, if we wanted to get the post with the `id` of `42` we would go to `/posts/42`. So you could create a new line in your routes file for each post...but that would get ridiculous and you would have to modify your web server every time someone posts. So! we have dynamic routes"